specification/dimensions/billingcurrency.md

```diff
@@ -0,0 +1,76 @@
+### Billing Currency
+
+Billing Currency is an identifier that represents the currency that a charge for resources and/or services was billed in. Billing Currency is commonly used in scenarios where costs need to be grouped or aggregated.
+
+The BillingCurrency column MUST be present in the billing data. BillingCurrency MUST match the currency used in the invoice generated by the invoice issuer. This column must be of type String and MUST NOT contain null values. BillingCurrency MUST be one of the values supported in [ISO 4217:2015](https://www.iso.org/standard/64758.html).
+
+[Attribute: Currency Code Format](link) contains further details about formatting requirements for currency values.
+
+Column ID
+
+BillingCurrency
+
+#### Display name
+
+Billing Currency
+
+#### Description
+
+Represents the currency that a charge was billed in.
+
+| Constraint      | Value                               |
+|-----------------|-------------------------------------|
+| Column required | True                                |
+| Data type       | String                              |
+| Allows nulls    | False                               |
+| Value format    | list-of-values                      |
+| Allowed Values  | Values supported in  ISO 4217:2015. |
```

Comment:
Feels like we reference our attribute here rather than the ISO. Our attribute could be more comprehensive than a single ISO standard. I think thats the generic way we want to do this for currency, date/time etc. where we have an attribute specifying a set of requirements.